docs/DeveloperGuide.adoc

```diff
@@ -12,9 +12,9 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103-F10-3/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Group F10-3`      Since: `Aug 2019`      Licence: `MIT`
 
 == Setting up
 
```

Comment:
For Figure 2 Class Diagram of the Logic Component, it would be better if the classes and arrows and the different sections can be color-coded to make it easier for visualisation.